hack/ovn-kind-cno.sh

```diff
@@ -88,7 +88,7 @@ if [ "$BUILD_OVN" = true ]; then
   pushd dist/images
   sudo cp -f ../../go-controller/_output/go/bin/* .
   cat << EOF | docker build -t origin-ovn-kubernetes:dev -f - .
-FROM quay.io/openshift/origin-ovn-kubernetes:4.3
+FROM quay.io/openshift/origin-ovn-kubernetes:4.4
```

Comment:
why not 4.5? not that it really matters.

Reply:
no strong reason, just that 4.4 is what is released